mera/utils.py

```diff
@@ -0,0 +1,81 @@
+from typing import Optional
+import pandas as pd
+import numpy as np
+
+
+def mask_too_few_records(
+    residual_df: pd.DataFrame,
+    ims: list,
+    event_id: Optional[str] = "event_id",
+    stat_id: Optional[str] = "stat_id",
+    min_num_records_per_event: Optional[int] = 3,
+    min_num_records_per_station: Optional[int] = 3,
```

Comment:
Per the [documentation](https://docs.python.org/3/library/typing.html#typing.Optional) on `Optional`

> Optional[X] is equivalent to X | None (or Union[X, None]).
> 
> Note that this is not the same concept as an optional argument, which is one that has a default. An optional argument with a default does not require the Optional qualifier on its type annotation just because it is optional.

In other words, only use `Optional` if `None` is a value you want to accept. In this case it looks like that would cause crashes to have `min_num_records_per_event` be `None`, for example.